mcp/README.md

```diff
@@ -207,10 +207,32 @@ attributes specific to only GAE standard
 | static_files.path | string | true within static_files context only | | none |
 | upload_path_regex | string |  true within static_files context only | | none |
 
+#### Manifest Files
```

Comment:
Manifest files are relevant for GAE deployments that specify a deployment type of `files`. This should be highlighted. They may be used for other services later but we'll see. There is also a Gradle plugin that will create this. Check in bitbucket for the repo.